app/setup.py

```diff
@@ -335,10 +335,10 @@ def setup_babel(application):
     def get_locale():  # pylint: disable=unused-variable
         session = get_session_store()
 
-        if not session:
-            return None
+        if session and session.session_data:
+            return session.session_data.language_code
 
-        return session.session_data.language_code
+        return None
```

Comment:
Ideally this would fallback to using HTTP `Accept-Language`, or some other flask-bable tool i think? because it looks like this would always render the error in english.

Reply:
I believe the intention is to default to english

Reply:
Oh okay, if that is the case I can approve this.

Reply:
Yes, expected behaviour is to default to English. Not sure if we would ever change that.